scripts/README.md

```diff
@@ -108,4 +108,105 @@ CMD=update_database python3 scripts/003_vlan_type_string.py
 ```
 `update_database` changes the value of every outdated TAG from integer to their respective string value.
 
-</details>
\ No newline at end of file
+<details><summary><h3>Redeploy symmetric UNI vlans EVPLs </h3></summary>
+
+[`redeploy_evpls_same_vlans.py`](./redeploy_evpls_same_vlans.py) is a CLI script to list and redeploy symmetric (same vlan on both UNIs) EVPLs.
+
+You should use this script if you want to avoid a redundant `set_vlan` instruction that used to be present in the instruction set. This script by triggering an EVC redeploy will force that all flows get pushed and overwritten again, it'll temporarily create traffic disruption. The redeploy in this case is just to force that the flows are pushed right away instead of waiting for a network convergence that might result in the flows getting pushed again.
+
+#### Pre-requisites
+
+- There's no additional dependency other than the existing core ones
```

Comment:
Nitpicking here but having `kytosd` running is a requisite. Before the scripts did not require it.

Reply:
Indeed @Alopalao, good observation. Added on https://github.com/kytos-ng/mef_eline/pull/390/commits/cdcda8f189f9e52673a7c30ddb8e4431d3a70782